examples/server-worker/index.ts

```diff
@@ -65,20 +80,20 @@ app.post('/key-shares/generate', async (req: Request, res: Response) => {
   const keyShares = new KeyShares();
   keyShares.update({ operators, publicKey });
 
-  // The nonce of the owner within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool
-  const TEST_OWNER_NONCE = 1;
   // The cluster owner address
   const TEST_OWNER_ADDRESS = '0x81592c3de184a3e2c0dcb5a261bc107bfa91f494';
```

Comment:
I don't know who put here test owner address but this is wrong. It should be also taken from post data because it's real working example, no stubs should be used